src/account.ts

```diff
@@ -4,6 +4,7 @@ export interface Account {
   params: {
     path: string;
     index: number;
+    currency?: string;
```

Comment:
why optional ?

Reply:
yes, you are right.
I've remove optional.